[2024-06-27T04:02:03.514+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T04:02:03.525+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-27T04:02:00.660093+00:00 [queued]>
[2024-06-27T04:02:03.530+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-27T04:02:00.660093+00:00 [queued]>
[2024-06-27T04:02:03.530+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-27T04:02:03.540+0000] {taskinstance.py:2330} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_to_gcs> on 2024-06-27 04:02:00.660093+00:00
[2024-06-27T04:02:03.549+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=318) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-27T04:02:03.549+0000] {standard_task_runner.py:63} INFO - Started process 320 to run task
[2024-06-27T04:02:03.547+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_transfer_to_gcs', 'upload_to_gcs', 'manual__2024-06-27T04:02:00.660093+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/to_gcs.py', '--cfg-path', '/tmp/tmpn0h_fg9m']
[2024-06-27T04:02:03.549+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask upload_to_gcs
[2024-06-27T04:02:03.594+0000] {task_command.py:426} INFO - Running <TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-27T04:02:00.660093+00:00 [running]> on host 5d6b262d6e21
[2024-06-27T04:02:03.666+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transfer_to_gcs' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T04:02:00.660093+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-27T04:02:00.660093+00:00'
[2024-06-27T04:02:03.667+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T04:02:03.681+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-06-27T04:02:03.685+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-06-27T04:02:17.245+0000] {gcs.py:560} INFO - File /opt/***/dags/data/exported_data.csv uploaded to data/exported_data.csv in testing-de bucket
[2024-06-27T04:02:17.246+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T04:02:17.261+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_transfer_to_gcs, task_id=upload_to_gcs, run_id=manual__2024-06-27T04:02:00.660093+00:00, execution_date=20240627T040200, start_date=20240627T040203, end_date=20240627T040217
[2024-06-27T04:02:17.307+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T04:02:17.327+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T04:02:17.329+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
